### PR TITLE
(DOCS-7779)(maint) Add redirects

### PR DESCRIPTION
**Pull request by hestonhoffman**
### Merge instructions

- [x] Please merge after reviewing

### CODE REVIEW

File: content/en/integrations/_index.md [hunk 1]
```diff
@@ -95,6 +95,23 @@ cascade:
     path: /integrations/feed.md
   aliases:
     - /integrations/rss/
+- _target:
+    path: /integrations/java.md
+  aliases:
+    - /agent/faq/jmx_integrations/
```
Comment (buraizu) [inline content/en/integrations/_index.md:101]: This one doesn't seem to work as expected

**Review by buraizu — APPROVED** [review]
Looks good overall and approved, but one redirect doesn't seem to work as expected and should be checked before merging

File: content/en/integrations/_index.md (hunk 1 above)
Comment (hestonhoffman) [reply content/en/integrations/_index.md:101]: The `jmx_integrations` one is from the ticket. It won't work until I merge the Dogweb one - the existence of an alias over there is causing a conflict. I'll make sure it works before I merge though 👍

File: local/bin/py/build/configurations/pull_config_preview.yaml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
 - config:
-    cache_enabled: true
+    cache_enabled: false
```
Comment (hestonhoffman) [inline local/bin/py/build/configurations/pull_config_preview.yaml:3]:
```suggestion
    cache_enabled: true
```
Switch back before merge

**Pull request merged**